server/model/orders.js

```diff
@@ -6,7 +6,7 @@ const orders = [
             {id:2,order:"Shawarma",quantity:4,price:4000},
```

Comment:
Please update the prices here to reflect the price of an individual item. The total is right.